src/sos/__main__.py

```diff
@@ -564,8 +563,7 @@ def cmd_run(args, workflow_args):
     try:
         # workflow args has to be in the format of --arg, not positional, not -a
         if workflow_args and not workflow_args[0].startswith("--"):
-            raise ValueError("Unrecognized command line option {}".format(
-                " ".join(workflow_args)))
+            raise ValueError(f"Unrecognized command line option {" ".join(workflow_args)}")
```

Comment:
This will not work due to the nested use of double quote. Use a different quote inside.